fluid-client/examples/preset-search-path.js

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env node
+
+const fluid = require('../src/fluidOsc');
+const FluidClient = require('../src/FluidClient');
+
+const client = new FluidClient(9999);
+client.send([fluid.plugin.addpath('/Users/zhiweigan/Library/Application Support/Tracktion/Waveform/Presets'),
```

Comment:
Again, watch out for checking absolute file paths. Remember you can use the '~' symbol, which will at least help things work on mac and linux.